server/router/profile.js

```diff
@@ -0,0 +1,56 @@
+const express = require('express');
+
+const Profile = require('../models/profileModel')
+
+const router = new express.Router();
+
+//An endpoint to fetch all of the user profiles
+router.get('/profiles', async (req,res) => {
+    try {
+        const profiles = await Profile.find();
+        res.json(profiles)
```

Comment:
can we add returns to these res.json / res.send?

Reply:
@meyeralex31  https://github.com/meyeralex31/csci550/pull/1/commits/d3776e95720c63e060e81d2879e4e94ec4b2d176